src/component/CreateList.jsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'react';
 import Table from './Table';
 import '../styles/createList.css';
+
 function CreateList() {
   const [create, setCreate] = useState('');
```

Comment:
if it is a list of items, why does the use state start with `''`?. useState can start with all types of data like [], {}, etc.

Reply:
because it is for input.Instead "save" is a list of items

Reply:
Is it wrong?